src/Core/Scheduler/Providers/VectorToColor_Provider.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Class: VectorToColor_Provider
+ * Description: Provides textures from a vector data
+ */
+
+
+import * as THREE from 'three';
+import togeojson from 'togeojson';
+import Extent from '../../Geographic/Extent';
+import GeoJSON2Texture from '../../../Renderer/ThreeExtended/GeoJSON2Texture';
+import GeoJSON2Three from '../../../Renderer/ThreeExtended/GeoJSON2Three';
+
+function getExtentFromGpxFile(file) {
+    const bound = file.getElementsByTagName('bounds')[0];
+    if (bound) {
+        const west = bound.getAttribute('minlon');
+        const east = bound.getAttribute('maxlon');
+        const south = bound.getAttribute('minlat');
+        const north = bound.getAttribute('maxlat');
+        return new Extent('EPSG:4326', west, east, south, north);
+    }
+    return new Extent('EPSG:4326', -180, 180, -90, 90);
+}
+
+function VectorToColor_Provider() {
```

Comment:
We started using the `class` syntax in other providers so you might want to use it here as well.

Reply:
Ok

Ps : There's no provider using class

Reply:
> Ps : There's no provider using class

Yeah.
But here's the process you're going to go through:
- convert `VectorToColor_Provider` to `class` syntax
- get errors from eslint because your methods won't be using `this`
- rewrite `VectorToColor_Provider` without `class` or `prototype` (see `PointCloudProvider`)

Reply:
done